agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -4837,14 +4837,19 @@ bool slave_thread::handle_ack_message(Socket *sd, ieee1905_1::CmduMessageRx &cmd
 bool slave_thread::handle_client_steering_request(Socket *sd, ieee1905_1::CmduMessageRx &cmdu_rx)
 {
     const auto mid = cmdu_rx.getMessageId();
-    LOG(DEBUG) << "Received CLIENT_STEERING_REQUEST_MESSAGE , mid=" << std::hex << int(mid);
 
     auto steering_request_tlv = cmdu_rx.addClass<wfa_map::tlvSteeringRequest>();
     if (!steering_request_tlv) {
         LOG(ERROR) << "addClass wfa_map::tlvSteeringRequest failed";
         return false;
     }
 
+    if (steering_request_tlv->bssid() != hostap_params.iface_mac) {
```

Comment:
@morantr Could you double-check if this is the correct parameter to use as bssid?

To be honest, I'm a bit confused about all the different terminology used in beerocks...

Reply:
slave_thread::handle_cmdu_control_message
...
    // to me or not to me, this is the question...
    if (beerocks_header->radio_mac() != hostap_params.iface_mac) {
        return true;
    }

Reply:
This is not correct.
The BSSID used in the message is actually a VAP (Virtual AP) mac which could be different from the radio mac.
Under the radio which transmits and lets stations to connect, could be VAPs which will operate on the same radio and will have the same channel and bandwidth, but have different SSID and credentials and MAC, of course.

You will have to pass it to ap_manager, which will check if it has the BSSID as one of its VAPS.

Reply:
Done in separate commit.